sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -42,6 +56,19 @@ typedef struct
   } _internal;
 } az_storage_blobs_blob_client;
 
+
+/**
+ * @brief Init a client with default options
+ *
+ * @param client blob_client to initialize
+ * @param url a url to a blob storage account
+ * @param credential credential object for authentication
+ *         #AZ_CREDENTIAL_ANONYMOUS should be used for SAS
+ * @param options client options
+ *
+ * @return An #az_result value indicating the result of the operation:
+ *         - #AZ_OK if successful
+ */
 AZ_NODISCARD az_result az_storage_blobs_blob_client_init(
     az_storage_blobs_blob_client* client,
     az_span uri,
```

Comment:
uri vs url - parameter vs comment mismatch, this is why CI is failing.

@JeffreyRichter what should the parameter name be? We use url in most places (az_http_transport, az_aad, az_credential_client_secret, az_http_request), but the keyvault and storage clients use uri.

Reply:
Actually, we should be using 'endpoint' everywhere.

Reply:
Is endpoint the azure sdk standard for all Uri & Url parameters?

Reply:
At one point the arch board agreed to use endpoint, see https://azure.github.io/azure-sdk/java_design.html and https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs#L63